app/views/CheckRegisteredDetailsView.scala.html

```diff
@@ -46,15 +46,22 @@
         @heading(messages("checkRegisteredDetails.heading", registeredDetails.eori))
         @warning(content=Html(messages("checkRegisteredDetails.warning")))
         @paragraph(Html(messages("checkRegisteredDetails.paragraph.1")))
+        @subheading(messages("checkRegisteredDetails.subheading.1"))
+
+        @if(registeredDetails.name.trim != "") {
+            @paragraph(content=Html("<strong>" + messages("checkRegisteredDetails.paragraph.2") + "</strong>"))
```

Comment:
I looks like these bold bits should have h3 types (headingH3 in the components package), may i suggest switching them out?

Reply:
Ah right! Perhaps this change to the component to give a custom size (like the **Heading** component does).


 ```
            @(message: String, size: String = "m")

             <h3 class="govuk-heading-@size">@message</h3>
```